tests/linux_kernel/helpers/test_mm.py

```diff
@@ -291,3 +292,7 @@ def test_environ(self):
             proc_environ = f.read().split(b"\0")[:-1]
         task = find_task(self.prog, os.getpid())
         self.assertEqual(environ(task), proc_environ)
+
+    @skip_unless_have_full_mm_support
+    def test_totalram_pages(self):
+        self.assertTrue(totalram_pages(self.prog) > 1000)
```

Comment:
I believe that the "MemTotal" line in `/proc/meminfo` comes from `totalram_pages()` in the kernel (see `meminfo_proc_show()` and `si_meminfo()`. So this test case can parse that file and compare the exact value.

Reply:
Yep, a nice suggestion.